easybuild/easyblocks/n/numpy.py

```diff
@@ -166,6 +166,21 @@ def get_libs_for_mkl(varname):
         if fft:
             extrasiteconfig += "\n[fftw]\nlibraries = %s" % fft
 
+        suitesparseroot = get_software_root('SuiteSparse')
+        if suitesparseroot:
+            amddir = os.path.join(suitesparseroot, 'AMD')
+            umfpackdir = os.path.join(suitesparseroot, 'UMFPACK')
```

Comment:
I mean to ask @sbecuwe if he's going to tackle my remark, since it's blocking this PR.